scripts/triggers-end-to-end-tests/firebase.json

```diff
@@ -0,0 +1,25 @@
+{
+  "database": {
+    "rules": "database.rules.json"
+  },
+  "firestore": {
+    "rules": "firestore.rules",
+    "indexes": "firestore.indexes.json"
+  },
```

Comment:
I assume we don't use firestore

Reply:
We are not, but this blob seems necessary for creating the firestore emulator client. Judging by the error I see when I remove this, creating a Firestore client taps into some pool of cached clients that (I assume) sources this config somewhere.

Reply:
This config is also used to populate the firebase-tools `EmulatorRegistry`, and the functions emulator thinks the Firestore emulator is not running if this entry doesn't exist.

Reply:
Yeah if you want `emulators:start` to start firestore you need a `firestore` block in your `firebase.json`. 

We could definitely remove the `indexes` sub-entry and the `indexes.json` file though.